packages/fast-css-editor-react/src/position/index.tsx

```diff
@@ -0,0 +1,173 @@
+import * as React from "react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import style, { ICSSPositionClassNameContract } from "./position.style";
+
+export enum PositionValue {
+    static = "static",
+    absolute = "absolute"
```

Comment:
are we currently only providing partial values here for a reason? Otherwise, I would expect all currently supported position values to be valid enums.

Reply:
they will be added as we decide which ones we will light up